dev-docs/bidders/openwebxchange.md

```diff
@@ -1,5 +1,7 @@
 ---
 layout: bidder
+deprecated: true
+notes: This adapter has been replaced by 'openweb'. Please use the 'openweb' bidder.
```

Comment:
```suggestion
pbjs_version_notes: This adapter has been replaced by 'openweb'. Please use the 'openweb' bidder.
```

There's currently support for a `deprecated` flag, even though I quite like it 😊